app/views/api/components/ApiData.vue

```diff
@@ -11,10 +11,16 @@
           th.month.border Month
           th.number.border License days used
           th.number.border Users with active licenses
+          th.number.border Lines of Code
+          th.number.border Programs wrote
```

Comment:
```suggestion
          th.number.border Programs written
```

Reply:
We may want to localize these; I know @smallst had a partner using this outside the US.